docs/dev/vagrant.rst

```diff
@@ -121,13 +130,14 @@ For instructions on contributing changes and our ``git`` workflow, see `<contrib
 Problems
 --------
 
-1. The ``vagrant up`` command errors out with a Ruby stack trace.
+1. The ``vagrant up`` command errors out, or times out while waiting for the VM to boot. (You may also want to investigate some of these for errors later in the process.)
+
+    If it errors out with a Ruby stack trace, there is a `known issue <https://github.com/mitchellh/vagrant/issues/6748>`_ with Vagrant/VirtualBox on IPv6 static networking.
```

Comment:
This might also be an issue if you accidentally installed 32-bit vagrant for some reason. Not sure if this is worth mentioning.